### PR TITLE
Swap scrollable tab bar with fixed width version

### PR DESCRIPTION
**Pull request by l2succes**

### CODE REVIEW

File: src/lib/Components/TabBar.tsx [hunk 1]
```diff
@@ -36,11 +38,19 @@ const Tabs = styled.View`
   border-bottom-width: 1px;
 `
 
-const TabButton = styled.View`
+const TabButton = styled.View<{ spaceEvenly?: boolean; active?: boolean }>`
```
Comment (ashfurrow) [inline src/lib/Components/TabBar.tsx]: 🙌

**Comment by orta** [comment]
This was tested to look good on the least-wide phone?

**Comment by ashfurrow** [comment]
@orta I can test now.

**Comment by ashfurrow** [comment]
Yeah this is looking wonky on iPhone SE:

Screen Shot 2019-03-15 at 12 08 59 PM
Screen Shot 2019-03-15 at 12 08 53 PM
Screen Shot 2019-03-15 at 12 09 18 PM

**Comment by ashfurrow** [comment]
Homepage looks good, though. I'll take a look.

**Comment by ashfurrow** [comment]
https://github.com/artsy/emission/pull/1514/commits/d6b9c48bccd5a7f88a1270d70747bef4af0edeb3 fixes the problem by having the tab buttons grow to fill their flex container:

Screen Shot 2019-03-15 at 12 52 12 PM
Screen Shot 2019-03-15 at 12 52 06 PM

I tested the Favourites and Home tabs too, and they look good. The Saves & Follows has a(n unrelated) bug where tapping/scrolling to "Shows" doesn't make the tab bar button totally visible. I will open a follow-up ticket.

**Comment by l2succes** [comment]
merge on green

**Pull request merged**